.github/workflows/ci.yml

```diff
@@ -46,7 +46,7 @@ jobs:
     - run:  npm ci
 
     - name: Linter
-      run:  npm run lint:ci
+      run:  npm run
```

Comment:
?

Reply:
Oh oops, I forgot to grab the second commit off the other PR.